docs/mast/mast.rst

```diff
@@ -4,6 +4,13 @@
 MAST Queries (`astroquery.mast`)
 ********************************
 
+Introduction
+============
+
+Astroquery.mast is a module available through the Astroquery Python toolkit. To 
+install Astroquery, follow the instructions detailed 
+`here <https://astroquery.readthedocs.io/en/latest/>`__.
+
```

Comment:
While the opening index page should be revamped (see e.g. https://github.com/astropy/astroquery/issues/2555)
Mast is not really a standalone thing in astroquery, so this introduction is somewhat out of the place, we don't provide install instructions on each subpage.

Reply:
We'd like to add a "basic overview" and description of the `mast` tool similar to what ESA JWST has [here](https://astroquery.readthedocs.io/en/latest/esa/jwst/jwst.html) for those who are new to the tool. Makayla has removed the installation instructions since those are already available in the Index, but we'd like to push this Introductions section to main, and perhaps we can move this blurb into the index once #2555 turns to in-progress.